.github/workflows/integration_tests.yml

```diff
@@ -32,22 +32,41 @@ jobs:
         uses: hashicorp/setup-terraform@v1
         with:
           terraform_version: 0.12.25
+          terraform_wrapper: false # wrapper would print out terraform command to stdout first
 
       - name: Install dependencies
         run: |
           sudo apt-get update
           sudo apt-get install -y jq
           sudo apt-get install -y pssh
+          sudo ln -s $(which parallel-ssh) /usr/local/bin/pssh
+          sudo ln -s $(which parallel-scp) /usr/local/bin/pscp
+
+      - name: Setup SSH
+        run: |
+          mkdir -p ~/.ssh/
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+          echo "${{ secrets.SSH_PUBLIC_KEY }}" > ~/.ssh/id_rsa.pub
```

Comment:
can you put the original input to the keybase team directory?

Reply:
It's already there, along with AWS credentials. `team/constellation_op/ci/`